### PR TITLE
Update donate contact label and desc

### PR DESCRIPTION
**Pull request by vilsbole**
[[ch8126]](https://app.clubhouse.io/mycrypto/story/8126/address-book-identicon-squished-w-long-label-note-update-tip-jar-copy)

1. Update default contact label and description
2. Limit column size to avoid squish
3. Add migration to update existing contact addresses

### CODE REVIEW

File: src/components/EditableText.tsx [hunk 1]
```diff
@@ -83,7 +84,7 @@ function EditableText({ saveValue, value, className, bold, truncate }: Props) {
   };
 
   return (
-    <Wrapper className={className}>
+    <Box variant="rowCenter" height="100%" {...props}>
```
Comment (FrederikBolding) [inline src/components/EditableText.tsx:87]: This breaks the alignment on `AccountList`
image

File: src/services/Store/store/persist.config.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import { values } from '@vendor';
+
+import { migrations } from './persist.config';
+
+describe('Persist Migrations', () => {
+  it('3: Updates the MYC_DONATE_ADDRESS in a users persistence layer', () => {
+    const migration = migrations['3'];
+    const contact = values(defaultContacts)[0];
+    const actual = migration({
+      [LSKeys.ADDRESS_BOOK]: [{ ...contact, label: 'Deprecated label' }]
```
Comment (FrederikBolding) [inline src/services/Store/store/persist.config.spec.tsx:12]: Can we test that this keeps other entries in the address book, just as a sanity check?

**Review by FrederikBolding — CHANGES_REQUESTED** [review]

File: src/services/Store/store/persist.config.spec.tsx (hunk 2 above)
Comment (vilsbole) [reply src/services/Store/store/persist.config.spec.tsx:12]: Updated.

File: src/components/EditableText.tsx (hunk 1 above)
Comment (vilsbole) [reply src/components/EditableText.tsx:87]: Fixed.

**Review by FrederikBolding — APPROVED** [review]

**Pull request merged**